Shared/Extensions/ArrayExtensions.swift

```diff
@@ -35,6 +35,13 @@ public extension Array {
         }
         return false
     }
+    
+    // Safe accessing
+    func at(_ index: Int) -> Element? {
```

Comment:
There is an extension that does this already I think.

`some_array[safe:index]` will do bounds checking.

Reply:
Ok removing it